### ISSUE TITLE
Inference time question

### ISSUE THREAD

**Issue by huang45** [open-mmlab/mmsegmentation#141]
Hi, what is the GPU used to test the inference time in the model zoo? I tested HRNetV2p-W18-Small model by 2080ti and only get ~0.3 seconds. Is that normal?

**Comment by York1996OutLook** [comment]
i got the same result:
out：
(open-mmlab) yaochunchun@dl-7047:~/mmsegmentation$ python tools/benchmark.py configs/fastscnn/fast_scnn_4x8_80k_lr0.12_cityscapes.py \
>     pretrained_models/fast_scnn_4x8_80k_lr0.12_cityscapes-cae6c46a.pth
2020-09-22 11:01:57,902 - mmseg - INFO - Loaded 500 images
Done image [50 / 200], fps: 15.14 img / s
Done image [100/ 200], fps: 15.09 img / s
Done image [150/ 200], fps: 15.08 img / s
Done image [200/ 200], fps: 15.02 img / s


and from the paper:
Input Size | Class |  FPS
 -- | --  | 
1024 × 2048 |  68.0  | 123.5
512 × 1024 |  62.8 |  285.8
256 × 512  | 51.9 |  485.4

**Comment by huang45** [comment]
tested for hrnet w18s (512*512) by benchmark.py and got following result (it looks better):

python tools/benchmark.py configs/hrnet/fcn_hr18s_512x512_160k_my.py checkpoints/fcn_hr18s_512x512_160k_my_save/iter_32000.pth 
2020-09-22 14:12:18,743 - mmseg - INFO - Loaded 601 images
Done image [50 / 200], fps: 48.45 img / s
Done image [100/ 200], fps: 48.42 img / s
Done image [150/ 200], fps: 48.39 img / s
Done image [200/ 200], fps: 48.34 img / s
Overall fps: 48.34 img / s

but ~0.16 seconds was showed during the training:

2020-09-22 11:28:45,243 - mmseg - INFO - Iter [50/80000]	lr: 1.199e-01, eta: 3:41:06, **time: 0.166,** data_time: 0.004, memory: 2135, decode.loss_seg: 0.5788, decode.acc_seg: 64.0334, aux_0.loss_seg: 0.2789, aux_0.acc_seg: 62.9780, aux_1.loss_seg: 0.3045, aux_1.acc_seg: 59.7900, loss: 1.1622

**Comment by xvjiarui** [comment]
Hi @huang45 
We benchmarked with V100 machines. Please refer to [here](https://github.com/open-mmlab/mmsegmentation/blob/master/docs/model_zoo.md#hardware) for hardware reference. 
As for the training, the time is not accurate at the beginning of training stage. There is a lot going on, e.g. creating worker process, finding the optimial cudnn algorithm for current hardware and input, etc.

**Comment by xvjiarui** [comment]
Hi @York1996OutLook 
Our implementation framework is not the same as the original paper. You are very welcomed to comment here if there is some better(faster) PyTorch implementation.

**Comment by huang45** [comment]
@xvjiarui Got it, thank you!
